makefiles/host/rootfs.mk

```diff
@@ -56,3 +56,15 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_PATH=./thirdparty/ \
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
+
+build/root.fs/opt/openocd: build/root.fs/.base_install
```

Comment:
why keep the source on the image? Wouldn't it be better to just install the compiled files?

Reply:
hm... i thought that it was handy to have the sources to mess around and i didnt really understand how to reliably do make install otherwise

Reply:
hmm, maybe.

As for `make install`, the `--prefix` option of `configure` should to the trick.